docs/sdk/client-side-sdks/javascript.md

```diff
@@ -85,15 +85,18 @@ To grab the value, there is a property on the object returned to grab the value:
 const value = variable.value
 ```
 
-If the value is not ready, it will return the default value passed in the creation of the variable. To get notified when the variable is loaded: 
+If the value is not ready, it will return the default value passed in the creation of the variable. 
+
+The `onUpdate` accepts a handler function that will be called whenever a variable value has changed.
+This can occur as a result of a project configuration change or calls to `identifyUser` or `resetUser`. There can only be one onUpdate function registered at a time, subsequent calls to this method will overwrite the previous handler:
```

Comment:
```suggestion
This can occur as a result of a project configuration change or calls to `identifyUser` or `resetUser`. There can only be one onUpdate function registered at a time. Subsequent calls to this method will overwrite the previous handler:
```